app/Http/Controllers/Playlist/PlaylistsController.php

```diff
@@ -273,35 +274,13 @@ private function createNewPlaylistItems($playlist, $playlist_items)
             $playlist_items = array_slice($playlist_items, 0, $allowed_size);
         }
 
-        $playlist_items_to_create = [];
-        $order = 1;
+        $playlist_items_object = [];
 
         foreach ($playlist_items as $playlist_item) {
-            $playlist_item = (object) $playlist_item;
```

Comment:
I have refactored this part because it was a similar code in the line: 705 in the same controller.

https://github.com/faithcomesbyhearing/dbp/pull/751/files#diff-b2074cf830f20db0d05674376fd81b979af9a990752eb8e202431ca6729db2e4L705